src/main/java/org/openstreetmap/atlas/geography/atlas/packed/PackedAtlasCloner.java

```diff
@@ -36,11 +41,24 @@ public PackedAtlas cloneFrom(final Atlas atlas)
     {
         final PackedAtlasBuilder builder = new PackedAtlasBuilder();
         builder.setSizeEstimates(atlas.metaData().getSize());
-        final AtlasMetaData metaData = atlas.metaData();
+        AtlasMetaData metaData = atlas.metaData();
         if (this.shardName != null)
         {
             metaData.copyWithNewShardName(this.shardName);
         }
+
+        if (this.additionalMetaDataTags.isPresent())
+        {
+            // Create a new map to avoid modifying the old atlas tags
```

Comment:
Nit: this comment no longer needed, since the new map is implicit in the getTags() call with the latest changes.

Reply:
great catch!